src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -62,11 +62,6 @@
     @ArgumentCollection
     protected IntervalArgumentCollection intervalArgumentCollection = requiresIntervals() ? new RequiredIntervalArgumentCollection() : new OptionalIntervalArgumentCollection();
 
-    @Argument(doc = "maximum number of bytes to read from a file into each partition of reads. " +
-            "Setting this higher will result in fewer partitions. Note that this will not be equal to the size of the partition in memory",
-            fullName = "bamPartitionSize", shortName = "bps", optional = true)
-    protected long bamPartitionSplitSize = ReadsSparkSource.DEFAULT_SPLIT_SIZE;
-
```

Comment:
I was talking about BQSR itself (ie., `BaseRecalibratorSpark`) rather than ApplyBQSR -- I'm concerned that the map phases there might run into memory issues with the larger partitions.

I think for now we should still expose this option so that we can have the ability to experiment during development (and when we run into memory issues). Once we all become confident that it's never needed, we can remove it at that point.